lib/pco_api/people/workflow/card/activity.ex

```diff
@@ -56,7 +56,7 @@ defmodule PcoApi.People.Workflow.Card.Activity do
 
   ## Example:
 
-      iex> %PcoApi.Record{type: "WorkflowCard", id: 1, links: %{"self" => "http://example.com"}} |> Address.get(2)
+      iex> %PcoApi.Record{type: "WorkflowCard", id: 1, links: %{"self" => "http://example.com"}} |> Activity.get(2)
```

Comment:
Let's be explicit about the module name in the docs. So `Activity` would become `Workflow.Card.Activity`.